app/assets/stylesheets/modules/sitter.scss

```diff
@@ -164,45 +202,108 @@
       font-size: $sitter-headline-font;
       font-weight: $sitter-headline-weight
     }
+  }
 
-    .hr {
-      opacity: 0.3;
-    }
+  .hr-1 {
+    width: 100%;
   }
 
- .price {
+  .hr {
+    width: 55%;
+    opacity: 0.3;
+    margin-top: 0;
+  }
+
+  .categories {
+    margin-bottom: 2em;
+    padding-bottom: 2em;
+  }
+
+  .price {
+    padding-top: 3em;
+
+    .small-3 {
+      @include category-details
+    }
+
     .small-9 {
       @include sitter-details;
     }
   }
 
   .availability {
+
+    .small-3 {
+      @include category-details
+    }
+
     .small-9 {
       @include sitter-details;
     }
   }
 
   .property-type {
+
+    .small-3 {
+      @include category-details
+    }
+
     .small-9 {
       @include sitter-details;
     }
   }
 
   .yard-type {
+
+    .small-3 {
+      @include category-details
+    }
+
     .small-9 {
       @include sitter-details;
     }
   }
 
   .have-dogs {
+
+    .small-3 {
+      @include category-details
+    }
+
     .small-9 {
       @include sitter-details;
     }
   }
 
   .have-children {
+
+    .small-3 {
+      @include category-details
+    }
+
     .small-9 {
       @include sitter-details;
     }
   }
 }
+
+.map {
+  padding-right: 2em;
+}
+
+#map-canvas1 {
+  margin-top: 1em;
```

Comment:
Properties should be ordered box-shadow, height, margin-bottom, margin-top, max-width